src/app/context-menu-builder.ts

```diff
@@ -433,6 +435,21 @@ export class ContextMenuBuilder {
     return menu;
   }
 
+  /**
+   * Adds the Paste menu item.
+   */
+  public addPasteAndMatchStyle(menu, menuInfo) {
```

Comment:
should this be renamed into addPasteAsPlainText? since the meaning of whole thing is for doing this

Reply:
depends if we're considering chromium definition or functional def.